src/main/java/com/flightstats/hub/webhook/InternalWebhookClient.java

```diff
@@ -22,17 +21,15 @@
 public class InternalWebhookClient {
     private final Client client;
     private final CuratorCluster hubCluster;
+    private final String uriScheme;
 
     @Inject
-    public InternalWebhookClient(@Named("HubCuratorCluster") CuratorCluster hubCluster) {
-        this.hubCluster = hubCluster;
-        this.client = RestClient.createClient(5, 15, true, true);
-    }
-
-    @VisibleForTesting
-    InternalWebhookClient(Client client, CuratorCluster hubCluster) {
+    public InternalWebhookClient(@Named("HubCuratorCluster") CuratorCluster hubCluster,
+                                 @Named("WebhookClient") Client client,
```

Comment:
If I'm not mistaken, I think this should be the constant you added up in the named bindings list.